### DISCUSSION TITLE
四川大学（江安校区）

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#5] [University]
> 回答来自群友，我仅负责搬运

## 大一大二
上床下桌，四人间，有空调，独卫

有早自习但你可以鸽子，无晨跑，有跑步打卡但可以一个舍友骑自行车带四个手机

允许外卖，不能进宿舍，但是出了宿舍门就能取

地铁…不知道。

洗衣房，一桶五元

校园网：
免费的限时，略微限速
十块钱一个月可以破时长
二十一个月可以加速
三十一个月可以同时满足上两项

十二点断电所以十二点之后还是没校园网
可以办流量卡二十多一个月，但是你会被广告短信骚扰

食堂对北方人不友好，重油重辣，纯素菜能压到五块钱以下，荤菜套餐十几块，炒菜大概二十多。
食堂土豆不削皮，土豆爱好者表示mmp

> cc by nc sa 3.0
> "131 的群"

**Comment by a5sl** [comment]
学校有两个地铁站，去市区约一小时。
校园网一个月免费100小时，你办校园电话卡，二三十块，应该会送宽带就可以用宽带。
洗衣服有丐点的选项，不洗那么久两三块吧。
夏天在校园里行走等于酷刑，有一条巨tm长的桥，没有任何挡阴。但有一说一校园是真挺漂亮的。
食堂纯猪食，喜欢川渝食物的才能勉强缓解。优点可能就是成本低了...
校外是郊区，离双流机场很近，吃的玩的不算很丰富，但至少台球、麻将馆管够。
